cmd/createorupdate/deploy.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/log"
+)
+
+func deploy(cs *api.OpenShiftManagedCluster, p api.Plugin, azuredeploy []byte) error {
+	var t map[string]interface{}
+	err := json.Unmarshal(azuredeploy, &t)
+	if err != nil {
+		return err
+	}
+
+	authorizer, err := auth.NewAuthorizerFromEnvironment()
```

Comment:
This would read config from azure.conf?

Reply:
this is mock implementation; in the RP it would use RP credentials